docs/src/storagext-cli/storage-provider.md

```diff
@@ -137,10 +137,10 @@ The `WINDOWED_POST` JSON object has the following structure:
 
 The `PROOF` JSON object has the following structure:
 
-| Name          | Description                                       |
-| ------------- | ------------------------------------------------- |
-| `post_proof`  | Proof type ("2KiB" or "StackedDRGWindow2KiBV1P1") |
-| `proof_bytes` | Hex encoded proof                                 |
+| Name          | Description                                                                                                                                           |
+| ------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `post_proof`  | Proof type ("2KiB", "8MiB", 512MiB, 1GiB, "StackedDRGWindow2KiBV1P1", "StackedDRGWindow8MiBV1", "StackedDRGWindow512MiBV1", "StackedDRGWindow1GiBV1") |
```

Comment:
Same as above, I think we should only allow for 1GiB here. 
I'd gate otlher sector sizes behind `devnet` feature or similar.
Gating for sure should be a a separate task.